packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -785,6 +785,7 @@ export class OdspDocumentStorageService implements IDocumentStorageService {
                 overalltime: overallTime,
                 networktime: networkTime,
                 clienttime: clientTime,
+                sltelemetry: response.headers.get("x-fluid-sltelemetry"),
```

Comment:
Can you please add some comment on what it is and where it is used?
I found myself couple times where there was a need (due to refactoring) to changing things, and it's impossible to say what would break if I change something.

Reply:
Added comment.